changelog/5957.feature.rst

```diff
@@ -0,0 +1,4 @@
+Add new entity extractor ``RegexEntityExtractor``. The entity extractor extracts entities using the lookup tables
+and regexes defined in the training data. You can either use the ``RegexEntityExtractor`` or the ``RegexFeaturizer``
+in your pipeline. If you use both components an error will be thrown.
```

Comment:
let's remove the error. Why not use both components?

Reply:
e.g. regexentityextractor with only lookup (see comment below) and actual regexes as features